src/components/Errors.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import { ErrorMessages } from '../enums/ErrorMessages';
+import { Dispatch, FC, SetStateAction, useEffect } from 'react';
+
+interface ErrorsProps {
+  errorMessage: ErrorMessages;
+  setErrorMessage: Dispatch<SetStateAction<ErrorMessages>>;
+}
+
+export const Errors: FC<ErrorsProps> = ({ errorMessage, setErrorMessage }) => {
+  useEffect(() => {
+    if (errorMessage) {
+      const timer = setTimeout(() => {
+        setErrorMessage(ErrorMessages.NO_ERRORS);
+      }, 3000);
+
+      return () => clearTimeout(timer);
+    }
+  }, [errorMessage]);
```

Comment:
GitGab Actions gives you this recomendation:
'React Hook useEffect has a missing dependency: 'setErrorMessage'. Either include it or remove the dependency array. If 'setErrorMessage' changes too often, find the parent component that defines it and wrap that definition in useCallback'
Check it in your code